src/main/java/org/opentripplanner/graph_builder/module/NearbyStopFinder.java

```diff
@@ -138,16 +139,16 @@ public NearbyStopFinder(Graph graph, double radiusMeters, boolean useStreets) {
     public List<StopAtDistance> findNearbyStopsViaStreets (
             Set<Vertex> originVertices,
             boolean reverseDirection,
-            boolean removeTempEdges
+            boolean removeTempEdges,
+            RoutingRequest routingRequest
```

Comment:
This breaks arriveBy-searches, as `orginVertices` get set to `from` on row 145. Either the direction should be always forward, or the originVertives should be set in `to` and not `from` parameter in `setRoutingContext `.

Reply:
I have investigated this. What we want is for the access search to always be a forward search, while the egress search should always be an arriveBy search, regardless of the main search being arriveBy or not. The direct street search, however, should be based on whether the main RoutingRequest is arriveBy or not.

I have pushed a fix with a TODO in the AccessEgressRouter. This gets a bit messy because we have not separated the main search from the street sub search, and it also ties into how the RoutingContext is created. It works now, cleaning up the request structure is something we should do soon.

Reply:
This seems to be fixed now